pywps/app/Process.py

```diff
@@ -162,26 +162,10 @@ def _set_uuid(self, uuid):
         for outpt in self.outputs:
             outpt.uuid = uuid
 
-    def _setup_status_storage(self):
-        self._status_store = StorageBuilder.buildStorage()
-
-    @property
-    def status_store(self):
-        if self._status_store is None:
-            self._setup_status_storage()
-        return self._status_store
-
-    @property
-    def status_location(self):
-        return self.status_store.location(self.status_filename)
-
-    @property
-    def status_filename(self):
-        return str(self.uuid) + '.xml'
```

Comment:
What if we want to keep this file available locally (on top of allowing `request=GetStatus`)? 
Can't the handlers be left available? 

The *OGC API - Processes* package I work on that uses PyWPS for corresponding WPS operations expects the XML files to be made available to users to help them debug why their process failed.

Reply:
Hello @fmigneault ,

The file can be retrieved with a quite easy GET request from uuid and the json data for generating the status is also stored within the database, thus I think this would be duplicated. I can implement it, but I would prefer that it is not implemented until the following comments get addresses.

Another thing is that currently the data storage within PyWPS seems to be a mess. The pywps.inout.storage does not seems to be used consistently across the current code. As far as remember the `FileStorage` must be used to ensure that the status file get a valid output url i.e. `[server] outputurl` configuration. And the `FileStorage` setup must be made in way that the path of stored file it the one exposed by `[server] outputurl`, thus I tryed to avoid that, that why I put the whole data within the database, avoiding using `FileStorage`.

The general idea here is that I would like to remove completly `[server] outputurl` and I would like every request goes to one entry point. that entry point will map request using the proper backend Storage facility. That way the storage facility and the way of the data are exposed is independent, and users are free to store data in the way they prefer by implementing they own `pywps.inout.Storage` or using one provided by pywps. For instance, this entry point may be a root path with several sub-path such as 'http://host/path/wps', 'http://host/path/status' or 'http://host/path/files' as I designed in the first place. We may also use GET parameters instead as I did in the current patch suite. I droped the path idea for GET method because the later is less invasive within the code and make the the setup of the server simpler. Note that I have another patch suite where I implement this behavior to store and access results data.

Best regards

Reply:
I agree that users could have the option to control where the status is stored and/or reported. However, to avoid breaking existing instances, it would be preferable if the default uses the local file as it was previously done. An option would have to be used to avoid producing the file. Any server that relies on this local XML file to exist locally will break.

Reply:
I will implement that option